Update or Remove Warning

dmauser opened the pull request:
Based on blog listed above there's no more that limitation. Certificates can be exported and used anywhere.

azurecla commented:
Hi __@raymondlaghaeian__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

dmauser committed:
Update or Remove Warning about

Based on blog listed above there's no more that limitation. Certificates can be exported and used anywhere.
